data/scripts.js

```diff
@@ -131,9 +131,8 @@ let BattleScripts = {
 	 * Dancer.
 	 */
 	useMove: function (move, pokemon, target, sourceEffect, zMove) {
```

Comment:
The easy solution (as described in my comment in #2367) was just to put `pokemon.moveThisTurnResult = undefined;` right here at the start of this function, but since I had already started to change the "hit a Protect-like move" value to `null` instead of `true` I just went ahead and did it the long way anyway.

Reply:
I actually do like that solution. Why did you change it?

Reply:
Is "hit Protect" notably different from a move success state in other ways?